### PR TITLE
optimize: add store mode,config type and registry type log info

### LINKED ISSUE #5209: Explicit output of storage mode information in logs
## Why you need it?
 Is your feature request related to a problem? Please describe in details
目前启动的时候没有明确输出seata-server使用的session和lock的type 如 db，redis，file
加上数据基本上是秒删，导致用户启动后并不是非常直观能知道自己配置的是否正确，需要在日志上增加标识信息

The current startup does not explicitly output the type of session and lock used by seata-server such as db, redis, file plus the data is basically deleted in seconds, resulting in the user not being able to intuitively know whether they are configured correctly after startup, and need to add identification information to the logs

## How it could be?
A clear and concise description of what you want to happen. You can explain more about input of the feature, and output of it.

### PR DESCRIPTION
**Pull request by renliangyu857**
…erver is starting

- [ ] I have registered the PR [changes](https://github.com/seata/seata/tree/develop/changes).

### Ⅱ. Does this pull request fix one issue?
fixes #5209

### CODE REVIEW

File: server/src/main/java/io/seata/server/ServerRunner.java [hunk 1]
```diff
@@ -47,6 +50,8 @@ public static void addDisposable(Disposable disposable) {
     public void run(String... args) {
         try {
             long start = System.currentTimeMillis();
+            LOGGER.info("seata server starting in config type:{},registry type:{},session store mode:{},lock store mode:{}",
```
Comment (a364176773) [inline server/src/main/java/io/seata/server/ServerRunner.java:53]: 这个地方加的不对，应该在configurationfactory和registryfactory里单独打印，这样client和server都能看到

Comment (renliangyu857) [reply server/src/main/java/io/seata/server/ServerRunner.java:53]: > 

好的

File: changes/zh-cn/develop.md [hunk 2]
```diff
@@ -31,6 +31,7 @@
 - [[#5177](https://github.com/seata/seata/pull/5177)] 如果 `server.session.enable-branch-async-remove` 为真，异步删除分支，同步解锁。
 - [[#5273](https://github.com/seata/seata/pull/5273)] 优化`protobuf-maven-plugin`插件的编译配置，解决高版本的命令行过长问题
 - [[#5303](https://github.com/seata/seata/pull/5303)] 移除启动脚本的-Xmn参数
+- [[#5325](https://github.com/seata/seata/pull/5325)] 当Server启动时添加配置中心、注册中心类型以及存储模式日志信息
```
Comment (a364176773) [inline changes/zh-cn/develop.md:34]: 不是当server，启动时，这里包括了client和server

File: server/src/main/java/io/seata/server/ServerRunner.java [hunk 3]
```diff
@@ -47,6 +50,7 @@ public static void addDisposable(Disposable disposable) {
     public void run(String... args) {
         try {
             long start = System.currentTimeMillis();
+            LOGGER.info("seata server starting in session store mode:{},lock store mode:{}", StoreConfig.getSessionMode(), StoreConfig.getLockMode());
```
Comment (a364176773) [inline server/src/main/java/io/seata/server/ServerRunner.java:53]: mode: {}, lock store mode: {}  逗号和冒号后面加空格，否则日志打出来不好看，日志放到server.start下面

File: config/seata-config-core/src/main/java/io/seata/config/ConfigurationFactory.java [hunk 4]
```diff
@@ -104,7 +104,7 @@ private static Configuration buildConfiguration() {
         String configTypeName = CURRENT_FILE_INSTANCE.getConfig(
                 ConfigurationKeys.FILE_ROOT_CONFIG + ConfigurationKeys.FILE_CONFIG_SPLIT_CHAR
                         + ConfigurationKeys.FILE_ROOT_TYPE);
-
+        LOGGER.info("use configuration center type:{}", configTypeName);
```
Comment (a364176773) [inline config/seata-config-core/src/main/java/io/seata/config/ConfigurationFactory.java:107]: 冒号后面加空格

File: discovery/seata-discovery-core/src/main/java/io/seata/discovery/registry/MultiRegistryFactory.java [hunk 5]
```diff
@@ -50,6 +54,7 @@ private static List<RegistryService> buildRegistryServices() {
         if (StringUtils.isBlank(registryTypeNamesStr)) {
             registryTypeNamesStr = RegistryType.File.name();
         }
+        LOGGER.info("use multi registry center type:{}", registryTypeNamesStr);
```
Comment (a364176773) [inline discovery/seata-discovery-core/src/main/java/io/seata/discovery/registry/MultiRegistryFactory.java:57]: 同上

File: discovery/seata-discovery-core/src/main/java/io/seata/discovery/registry/RegistryFactory.java [hunk 6]
```diff
@@ -43,6 +47,7 @@ private static RegistryService buildRegistryService() {
         String registryTypeName = ConfigurationFactory.CURRENT_FILE_INSTANCE.getConfig(
             ConfigurationKeys.FILE_ROOT_REGISTRY + ConfigurationKeys.FILE_CONFIG_SPLIT_CHAR
                 + ConfigurationKeys.FILE_ROOT_TYPE);
+        LOGGER.info("use registry center type:{}", registryTypeName);
```
Comment (a364176773) [inline discovery/seata-discovery-core/src/main/java/io/seata/discovery/registry/RegistryFactory.java:50]: 同上

**Review by a364176773 — APPROVED** [review]
LGTM

File: discovery/seata-discovery-core/src/main/java/io/seata/discovery/registry/MultiRegistryFactory.java [hunk 7]
```diff
@@ -50,6 +54,7 @@ private static List<RegistryService> buildRegistryServices() {
         if (StringUtils.isBlank(registryTypeNamesStr)) {
             registryTypeNamesStr = RegistryType.File.name();
         }
+        LOGGER.info("use multi registry center type: {}", registryTypeNamesStr);
```
Comment (l81893521) [inline discovery/seata-discovery-core/src/main/java/io/seata/discovery/registry/MultiRegistryFactory.java:57]: Why not print the registryTypeNames?

**Review by l81893521 — APPROVED** [review]
LGTM

File: server/src/main/java/io/seata/server/coordinator/DefaultCoordinator.java [hunk 8]
```diff
@@ -337,7 +338,8 @@ protected void timeoutCheck() {
                     return false;
                 }
 
-                LOGGER.warn("Global transaction[{}] is timeout and will be rollback,transaction begin time:{} and now:{}", globalSession.getXid(), globalSession.getBeginTime(), System.currentTimeMillis());
+                LOGGER.warn("Global transaction[{}] is timeout and will be rollback,transaction begin time:{} and now:{}", globalSession.getXid(),
```
Comment (slievrly) [inline server/src/main/java/io/seata/server/coordinator/DefaultCoordinator.java:341]: The annotation timeout is in milliseconds, here the log format is printed in seconds.

**Pull request merged**